.fides/fides.toml

```diff
@@ -49,3 +49,8 @@ task_retry_delay = 1
 
 [admin_ui]
 enabled = true
+
+[celery]
+event_queue_prefix = "fides_worker"
+task_default_queue = "fides"
+task_always_eager = true
```

Comment:
Can you explain these changes - getting rid of the celery.toml and adding this to the config?

Reply:
@pattisdr this was an additional upstream PR, having `celery.toml` as a separate config file didn't feel like a good UX, so now the user can directly define a `celery` subsection in their `fides.toml` and it will get passed through as expected. Additionally it is not validated, so the user can pass through whatever they want (same functionality as before, just in a single location)

Reply:
thanks